RustFxA/FxAWebViewController.swift

```diff
@@ -73,11 +65,8 @@ class FxAWebViewController: UIViewController, WKNavigationDelegate {
         webView.navigationDelegate = self
         view = webView
         
-        viewModel.authenticate()
-        
-        viewModel.onEmittingNewState = { [weak self] output in
-            let (request, method) = output
-            if let method = method {
+        viewModel.setupFirstPage { [weak self] (request, telemetryEventMethod) in
```

Comment:
Notes: `authenticate` isn't accurate, `setupFirstPage` is, and a completion handler is much simpler